configs/siklu_80X0_defconfig

```diff
@@ -79,13 +85,3 @@ BR2_PACKAGE_TAR=y
 BR2_TARGET_ROOTFS_SQUASHFS=y
 BR2_TARGET_ROOTFS_SQUASHFS4_LZMA=y
 BR2_TARGET_ROOTFS_TAR_GZIP=y
-BR2_PACKAGE_LIBCURL=y
-BR2_PACKAGE_LIBCURL_CURL=y
-BR2_PACKAGE_LIBCURL_PROXY_SUPPORT=y
-BR2_PACKAGE_LIBCURL_COOKIES_SUPPORT=y
-BR2_PACKAGE_LIBCURL_EXTRA_PROTOCOLS_FEATURES=y
-BR2_PACKAGE_LIBCURL_OPENSSL=y
```

Comment:
Why remove BR2_PACKAGE_LIBCURL_OPENSSL?

Reply:
I didn't remove it, if you rum make menuconfig you will see it 'y'

Reply:
Thy are hided because other components are included and include them

Reply:
can you please confirm that webUI is working after the change? (note: web UI uses OpenSSL package).

Reply:
@dima1308 web worked ok after my changes